src/widget/wvumeter.cpp

```diff
@@ -165,24 +166,31 @@ void WVuMeter::paintEvent(QPaintEvent *) {
     QStyleOption option;
     option.initFrom(this);
     QStylePainter p(this);
+    p.setBrush(option.palette.text());
     p.drawPrimitive(QStyle::PE_Widget, option);
-
+    
+    QRect contentRect = style()->subElementRect(QStyle::SE_FrameContents, &option, this);
+    if (contentRect.isNull()) {
+        contentRect = rect();
+    }
+    
     if (!m_pPixmapBack.isNull() && !m_pPixmapBack->isNull()) {
         // Draw background. DrawMode takes care of whether to stretch or not.
-        m_pPixmapBack->draw(rect(), &p);
+        m_pPixmapBack->draw(contentRect, &p);
     }
-
+    
```

Comment:
Please configure your editor, that it removes tailing white space or at least not introduces some.